docs/setup-and-installing.md

```diff
@@ -87,6 +91,8 @@ Once you have installed [Rider](https://www.jetbrains.com/rider/), install the [
 
 <%(/Expandable)%>
 
+<%(Callout type="alert" message="You need select Linux Build Support as in your setup because all server-workers in a cloud deployment run in a Linux environment. When you come to build your project, in the `Assets/Fps/Config/BuildConfiguration`, do not change the `UnityGameLogic Cloud Environment` from Linux.")%>
```

Comment:
As above, mention you need the Unity Installer for this option to appear etc etc